example/example.dart

```diff
@@ -12,7 +12,7 @@ Future<void> main() async {
   final pureApi = nh.API();
 
   // API client with HTTP proxy.
-  final proxyApi = NHentai.proxy(
+  final proxyApi = NHentaiAPI.proxy(
```

Comment:
This was intended name, you can have alias if you want, but main class should stay as it.